### PR TITLE
Admin Scholarship Dashboard View

### PR DESCRIPTION
**Pull request by BrodrickS**
For Task: https://trello.com/c/WsCKgsvF

### CODE REVIEW

File: frontend/src/Admin/components/NavBar.js [hunk 1]
```diff
@@ -49,6 +49,32 @@ export default class NavBar extends Component {
           <NavbarToggler onClick={this.toggle} />
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav className='ml-auto' navbar>
+              <NavItem>
+                <NavLink tag={Link} to='/'>
+                  Dashboard
```
Comment (JohnPMurray) [inline frontend/src/Admin/components/NavBar.js]: I would say change this to shows dashboard or something

File: frontend/src/Admin/components/PortfolioCard.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
+import React, { Component, Fragment } from 'react'
```
Comment (JohnPMurray) [inline frontend/src/Admin/components/PortfolioCard.js]: How much of this file was copied from shows card? Is there a way we can use any shared components?
For the student version of this I ended up turning some of the matching internals into components that they both used.

File: frontend/src/Admin/queries/portfolioPeriod.graphql [hunk 3]
```diff
@@ -0,0 +1,71 @@
+query PortfolioPeriod($id: ID!) {
+  show(id: $id) {
```
Comment (johnbyrne727rit) [inline frontend/src/Admin/queries/portfolioPeriod.graphql]: Is this pulling shows from the BE?

Comment (johnbyrne727rit) [reply frontend/src/Admin/queries/portfolioPeriod.graphql]: Discussed irl, should be portfolioPeriod now.

**Review by JohnPMurray — CHANGES_REQUESTED** [review]
Make sure to actually pull portfolio periods. Also I made a few comments to adress

**Comment by JohnPMurray** [comment]
I know you didn't make the create a portfolio period page, but we should change it to link to the portfolios dashboard (rather than the shows dashboard) when its done

**Comment by BrodrickS** [comment]
> I know you didn't make the create a portfolio period page, but we should change it to link to the portfolios dashboard (rather than the shows dashboard) when its done

Done

File: frontend/src/Admin/components/NavBar.js (hunk 1 above)
Comment (BrodrickS) [reply frontend/src/Admin/components/NavBar.js]: Changed to "Shows Dashboard"

File: frontend/src/Admin/components/PortfolioCard.js (hunk 2 above)
Comment (BrodrickS) [reply frontend/src/Admin/components/PortfolioCard.js]: Just got it working. Most of it is duplicated. The "shows" card has some additional functionality, I'll take a look at reusing components.

**Review by johnbyrne727rit — APPROVED** [review]
Looks groovy to me

**Review by JohnPMurray — APPROVED** [review]
looks good

**Pull request merged**